qubes_config/global_config/global_config.py

```diff
@@ -469,6 +469,11 @@ def _reset(self, _widget=None):
             page.reset()
 
     def _quit(self, _widget=None):
+        ''' Hide the main Window '''
+        self.main_window.hide()
+        while Gtk.events_pending():
+            Gtk.main_iteration()
+        ''' Then wait for disposable helper threads to finish '''
```

Comment:
```suggestion
        # Then wait for disposable helper threads to finish
```

Pylint dislikes this sort of comments - the triple-quotes should only be used for docstrings at the start of the method/class/whatnow, otherwise `#` should be used. (the same comment applies to the previous comment, it wasn't flagged by pylint because it's at the start of the function, so it thought it was a docstring describing the function. 

If you care about backstory for this, it's in https://peps.python.org/pep-0257/ :)

Reply:
> If you care about backstory for this, it's in https://peps.python.org/pep-0257/ :)

Thank you very much for the link. I will read it. I just changed the comments to start with # and forced pushed the amended commit.